docs/book/src/tasks/experimental-features/cluster-classes.md

```diff
@@ -0,0 +1,497 @@
+# Cluster Class
```

Comment:
Can you reference this file in the respective section in SUMMARY.md? Otherwise the doc is not easily accessible in the book.

Reply:
Also to `experimental-features.md`